epam-promo/components/tables/ColumnHeaderDropdown/ColumnHeaderDropdown.tsx

```diff
@@ -4,7 +4,7 @@ import { Modifier } from "react-popper";
 import { DropdownBodyProps, IDropdownToggler, isMobile, mobilePopperModifier } from '@epam/uui-core';
 import { Dropdown } from '@epam/uui-components';
 import { Panel } from '../../layout';
-import { MobileDropdownWrapper } from "../../pickers";
+import { MobileDropdownWrapper } from "@epam/uui";
 import { SortingPanel, SortingPanelProps } from "./SortingPanel";
 
 type ColumnHeaderDropdownProps = SortingPanelProps & {
```

Comment:
may be move ColumnHeaderDropdown in UUI also?

Reply:
It's next iterations. The transfer will be as part of updating the tables